playbooks/deploy_user.yml

```diff
@@ -1,10 +1,19 @@
 ---
-# Only run this playbook on all hosts in a maintenance window, because no one can deploy while the playbook is running.
```

Comment:
Which is / was the step that makes it so that no one can deploy? Is there a version of this playbook the deletes the authorized keys file altogether or something?

Reply:
I'm not sure. We added the warnings and the `serial` setting last year, before we changed the way we started using a template to manage authorized keys:
Warnings in May 2022: #3030
Serial in June 2022: #3094
Template in October 2022: #3377
Before October, we used the [authorized_key module](https://docs.ansible.com/ansible/latest/collections/ansible/posix/authorized_key_module.html#ansible-collections-ansible-posix-authorized-key-module) to manage keys.

My instinct would be to test the updates on a small group of staging servers and see how it behaves.